allel/util.py

```diff
@@ -145,6 +146,50 @@ def ensure_square(dist):
     return dist
 
 
+def mask_inaccessible(is_accessible, ac, pos, start=None, stop=None):
```

Comment:
Not currently using `start` and `stop`?

Also, small thought, this could be generalised to:

```suggestion
def mask_inaccessible(is_accessible, pos, *arrays, start=None, stop=None):
```

...allowing for masking any arrays, not just `ac`.